20xt/src/baja_ipc/include/live_comm_factory.h

```diff
@@ -0,0 +1,26 @@
+#ifndef LIVE_COMM_FACTORY_H
```

Comment:
I would actually like to remove this. I think it's unnecessary complexity for marginal performance gains

Reply:
so interestingly nothing in ros seems to have header guards

Reply:
I meant the whole file lol